### PR TITLE
Add multiple interactive examples

### PR DESCRIPTION
**Pull request by NiedziolkaMichal**
#### Summary
This commit contains multiple interactive examples that were commited recently.

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by NiedziolkaMichal** [comment]
Height of before & after pseudo elements should be fixed now. I see that contain & unicode-bidi return 404 now, but it's because they were just commited to interactive examples: [#2133](https://github.com/mdn/interactive-examples/pull/2133) [#2128](https://github.com/mdn/interactive-examples/pull/2128)

**Comment by NiedziolkaMichal** [comment]
I added [text-emphasis-style](https://developer.mozilla.org/en-US/docs/Web/CSS/text-emphasis-style) & [text-emphasis-color](https://developer.mozilla.org/en-US/docs/Web/CSS/text-emphasis-color) too.

**Comment by NiedziolkaMichal** [comment]
I added [text-underline-offset](https://developer.mozilla.org/en-US/docs/Web/CSS/text-underline-offset) too.

**Comment by NiedziolkaMichal** [comment]
also [text-decoration-thickness](https://developer.mozilla.org/en-US/docs/Web/CSS/text-decoration-thickness)

File: files/en-us/web/css/_doublecolon_after/index.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ browser-compat: css.selectors.after
 
 In CSS, **`::after`** creates a [pseudo-element](/en-US/docs/Web/CSS/Pseudo-elements) that is the last child of the selected element. It is often used to add cosmetic content to an element with the {{CSSxRef("content")}} property. It is inline by default.
 
+{{EmbedInteractiveExample("pages/tabbed/pseudo-element-after.html", "tabbed-standard")}}
+
 ```css
```
Comment (wbamberg) [inline files/en-us/web/css/_doublecolon_after/index.md:19]: I think we should remove this little syntax block now we have an interactive example.

File: files/en-us/web/css/_doublecolon_before/index.md [hunk 2]
```diff
@@ -14,6 +14,8 @@ browser-compat: css.selectors.before
 
 In CSS, **`::before`** creates a [pseudo-element](/en-US/docs/Web/CSS/Pseudo-elements) that is the first child of the selected element. It is often used to add cosmetic content to an element with the {{cssxref("content")}} property. It is inline by default.
 
+{{EmbedInteractiveExample("pages/tabbed/pseudo-element-before.html", "tabbed-standard")}}
+
 ```css
```
Comment (wbamberg) [inline files/en-us/web/css/_doublecolon_before/index.md:19]: And also here.

**Review by wbamberg — COMMENTED** [review]
A couple of little suggestions, but these are great. Awesome work @NiedziolkaMichal .

File: files/en-us/web/css/_doublecolon_after/index.md (hunk 1 above)
Comment (NiedziolkaMichal) [reply files/en-us/web/css/_doublecolon_after/index.md]: I wasn't sure what to do about those. Thank you, that's a good suggestion.

**Review by wbamberg — APPROVED** [review]
👍 thank you @NiedziolkaMichal !

**Pull request merged**